src/manage/invitations/response.rs

```diff
@@ -1,4 +1,4 @@
-//! Deepgram invitations API response types.
+//! Reponse module
```

Comment:
Documentation comments are public facing.  This for instance shows up here: https://docs.rs/deepgram/latest/deepgram/invitations/response/index.html

Can we give them a little love?  In this particular case, the original comment was more informative to a user than the new one (and didn't have a typo!)